dulwich/tests/test_bundle.py

```diff
@@ -18,20 +21,13 @@
 # License, Version 2.0.
 #
 
-"""Tests for bundle support."""
-
 import os
 import tempfile
 
-from dulwich.tests import (
-    TestCase,
-)
-
-from dulwich.bundle import (
-    Bundle,
-    read_bundle,
-    write_bundle,
-)
+from dulwich.bundle import Bundle
```

Comment:
See above - please don't reformat these imports. They're like this intentionally.

Reply:
OK